css/style.css

```diff
@@ -16,4 +15,149 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.list {
+  padding-right: 50px;
+}
+
+li {
+  display: inline;
+  padding: 15px;
+  font-size: 20px;
+  color: dimgray;
+  transition: font-size 0.5s ease;
+}
+
+li:hover {
+  color: red;
+  font-size: 25px;
+}
+
+.logo {
+  height: 50px;
+  width: 100px;
+  padding-left: 50px;
+  padding-top: 25px;
+  padding-bottom: 25px;
+}
+
+.backgroundImg {
+  background-image: url(../img/first-background.jpg);
+  background-size: cover;
+  background-position: center center;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: white;
+}
+.buttonS {
+  background-color: rgb(224, 65, 65);
+  padding: 20px 35px;
+  margin-top: 30px;
+  border-radius: 5%;
+  font-size: 20px;
+  border: none;
+}
+
+.buttonS:hover {
+  background-color: lightblue;
+}
+
+.firstText {
+  font-size: 70px;
+  font-weight: lighter;
+}
+.secondText {
+  font-size: 40px;
+  font-weight: lighter;
+}
+body {
+  text-align: center;
+}
```

Comment:
Avoid adding styles like this to the entire body. You should add it more locally to specific elements.